uriget/uriget.go

```diff
@@ -233,3 +239,38 @@ func (o *options) getGit(ctx context.Context, u *url.URL) ([]byte, error) {
 	o.logger.Printf("Read %d bytes from %s", len(buff), filepath.Join(td, subPath))
 	return buff, nil
 }
+
+func (o *options) getOci(ctx context.Context, u *url.URL) ([]byte, error) {
```

Comment:
I have a question: should we also support url with user and password or don't we foresee this use case? cc @mathieu-benoit

Reply:
Good question, for now with the other supported format, it's all public for now, so as part of this PR, it's just supporting public OCI images/registry too. We will need to file different issues/feature requests per format where we want to support private repo, private registry, etc.